src/storage/heap_file.h

```diff
@@ -309,6 +309,7 @@ struct heap_operation_context
   bool is_redistribute_insert_with_delid;	/* true if the insert is due to a partition redistribute data operation
 						 * and has a valid delid */
   bool is_bulk_op;		// note - currently for insert only
+  bool use_bulk_logging;	// note - currently for bulk insert only
   // side-effect - disables MVCC operations
```

Comment:
// side-effect - ...
this should be appended right after `is_bulk_op`. 

A horizontal blank before `use_bulk_logging` would help.

Reply:
fixed